cmd/influx/cli/cli.go

```diff
@@ -198,9 +199,20 @@ func (c *CommandLine) Run() error {
 	c.Version()
 
 	// Only load/write history if HOME environment variable is set.
+	var historyDir string
 	if homeDir := os.Getenv("HOME"); homeDir != "" {
-		// Attempt to load the history file.
-		c.historyFilePath = filepath.Join(homeDir, ".influx_history")
+		historyDir = homeDir
+	}
+
+	if runtime.GOOS == "windows" {
```

Comment:
Can you make it so that `USERPROFILE` doesn't get read if `HOME` is set? That way, if both `HOME` and `USERPROFILE` are set, then it uses `HOME`.